piker/ui/_style.py

```diff
@@ -99,6 +113,14 @@ def configure_to_dpi(self, screen: QtGui.QScreen | None = None):
         listed in the script in ``snippets/qt_screen_info.py``.
 
         '''
+        if self._font_size is not None:
+            self._set_qfont_px_size(self._font_size)
+            return
+
+        # NOTE: if no font size set either in the [ui] section of the
```

Comment:
maybe document this impl detail: the dynamic size calculating by DPI, in the class doc string?